src/FffGcodeWriter.cpp

```diff
@@ -778,6 +778,7 @@ void FffGcodeWriter::processSkin(GCodePlanner& gcode_layer, SliceMeshStorage* me
         if (bridge > -1)
         {
             pattern = EFillMethod::LINES;
+            skin_angle = bridge;
```

Comment:
Thank you very much.
We didn't think it was a bug; we thought the bridging algorithm was just really poor.
It's a good thing you noticed this!